lib/cList.ml

```diff
@@ -78,6 +78,7 @@ sig
   exception IndexOutOfRange
   val goto : int -> 'a list -> 'a list * 'a list
   val split_when : ('a -> bool) -> 'a list -> 'a list * 'a list
+  val split_with : ('a -> bool) -> 'a list -> 'a list * 'a list
```

Comment:
As I told you before, the standard name for this function is not `split_with` it's `partition` and it's already defined in [the standard library of OCaml](https://caml.inria.fr/pub/docs/manual-ocaml/libref/List.html#VALpartition). Any reason to redefine this function then?